src/events.cairo

```diff
@@ -1,3 +1,11 @@
+#[derive(Drop, starknet::Event)]
+struct Emote {
+    #[key]
+    game_id: u32,
+    player_index: u32,
+    emote: u32
```

Comment:
I think emote should be a `felt252` if you expect it to be a free string, otherwise it should be an `enum` with specified emotes.

Personal opinion: free string is fine.